core/src/test/java/io/questdb/griffin/engine/functions/groupby/AvgIntVecGroupByFunctionFactoryTest.java

```diff
@@ -76,6 +80,10 @@ public void testAllNullThenOne() throws Exception {
 
     @Test
     public void testSimple() throws Exception {
+        // fix page frame size, because it affects AVG accuracy
+
+        pageFrameMaxRows = 10_000;
```

Comment:
Same question as in `AvgDoubleVecGroupByFunctionFactoryTest`

Reply:
when default changes we do not want to be fixing broken tests

Reply:
If that's the motivation, better to do it in a `@BeforeClass` setup method rather than have these assignments in individual tests.